source/customizing_look_and_feel/04_configuring_data_visualizations/basemaps.rst

```diff
@@ -1,39 +1,166 @@
+2. In the Basemap column, click on the drop-down menu. By default, it should display "Streets" (default basemap for all OpenDataSoft domains).
+3. The drop-down menu indicates all basemaps available in the domain, ordered by basemap providers. Click on the basemap to add to the domain.
+4. In the Label column, indicate a label for the newly added basemap. There must be one label per language (if the domain supports more than one), otherwise the configurations cannot be saved.
+5. Click on the "Save" button in the top right corner of the page.
+
+
+Adding custom basemaps
+----------------------
+
+Among the default basemaps, displayed in the drop-down menu of the Basemap table column (described in the introduction of this documentation), are 2 options that allow the addition of custom basemaps:
+
+- "Custom", to add TMS and other OpenStreeMap compatible basemaps
+- "Custom (WMS)", to add WMS basemaps
+
+.. admonition:: Note
+   :class: note
+
+   WMTS basemaps are not supported by the OpenDataSoft plateform.
+
+To add a custom basemap to the domain:
+
+1. Click on the "Add basemap" button.
+2. In the Basemap column of the newly added row of the table, click on the drop-down menu.
+3. In the drop-down menu, select either "Custom" or "Custom (WMS)".
+4. Fill up the basemap configurations.
+
+* If adding a "Custom" basemap, please refer to the table below:
+
+  .. list-table::
+    :header-rows: 1
+
+    * * Name
+      * Description
+      * Example
+    * * Tiles URL (required)
+      * URL of the basemap.
+      * ``http://tile.basemapprovider.io/{z}/{x}/{y}.png``
+    * * Subdomains
+      * Subdomains of the basemap (provided by the basemap provider).
+      * abcd
+    * * Zoom min
+      * Minimum zoom available (the most global view we can have of the map).
+      * 1
+    * * Zoom max
+      * Maximum zoom available (the closer we can be into the map).
+      * 21
+    * * Attribution
+      * Source of the basemap data. Will be displayed at the bottom of the basemap everywhere it is used.
+      * Basemap data from OpenStreetMap
+    * * TMS service
+      * Should only be checked in case the basemap is a strict TMS basemap, with inverted coordinates.
+      *
+
+* If adding a "Custom (WMS)" basemap, please refer to the table below:
+
+  .. list-table::
+    :header-rows: 1
 
-    .. image:: images/basemaps__mapbox-integration--fr.png
-       :alt: Configuration de l'intégration Mapbox
+    * * Name
+      * Description
+      * Example
+    * * WMS URL (required)
+      * URL of the basemap.
+      * ``https://www.basemapprovider.com/wms``
+    * * Attribution
+      * Source of the basemap data. Will be displayed at the bottom of the basemap everywhere it is used.
+      * Basemap data from OpenStreetMap
+    * * Layers (required)
+      * Layer(s) of the basemap (provided by the basemap provider).
+      * LayerName
+    * * Tiles images format
+      * Format of the tiles: either .png or .jpeg (provided by the basemap provider).
```

Comment:
the usage is more .jpg than .jpeg

Reply:
WMS requires "image/jpeg" so I suppose it is better to keep the same writing as the one the WMS service may expose (e.g. your IT gives you the set of parameters to pass)

Reply:
".jpeg" is also what is used on the interface, that's why I used it rather than ".jpg".